Dockerfile

```diff
@@ -20,7 +19,7 @@ RUN mkdir -p /run/nginx
 COPY nginx.conf /etc/nginx/
 
 # copy swagger files to the `/js` folder
-ADD ./dist/* /usr/share/nginx/html/
+ADD ./dist/* /usr/share/nginx/html/${BASE_PATH}
```

Comment:
Now that I'm looking at this, I don't think this will work if the user wants to set `BASE_PATH` to anything else, since this happens during `docker build` instead of `docker run`.

So if I wanted to set `BASE_PATH` to something like "ui", I wouldn't be able to do that when I run the docker container like so: 

```
docker run -p 80:8080 -e BASE_PATH=/ui -e SWAGGER_JSON=/swagger/schema.json -v $(pwd)/schema:/swagger swaggerapi/swagger-ui
```

...because the files would still be located in `/usr/share/nginx/html/swagger`, which happened when the container was built.

I'm going to open a PR that fixes this.